### DISCUSSION TITLE
Lidarr support?

### DISCUSSION THREAD

**Discussion by oOStroudyOo** [Feramance/qBitrr#77] [Ideas]
Hello,

I've been looking for an app to manage the stalled downloads, so this seems great.
However, almost all of my stalled downloads tend to come from Lidarr. Are there any plans to support it?

Thanks.

**Comment by Feramance** [comment]
Hi, 

I'll take a look at the api when I get the chance to and I'll let you know whether I see this as worth it for this version or not. I'll work to implement it in the New version being developed.
